hubconf.py

```diff
@@ -130,7 +115,21 @@ def train(self, niter=1):
 
 if __name__ == '__main__':
     m = Model(device='cuda', jit=False)
+    t2 = time.time()
     model, example_inputs = m.get_module()
-    model(*example_inputs)
-    m.train()
-    m.eval()
+    t3 = time.time()
+
+    model(*example_inputs[:2])
```

Comment:
We'll run `model(*example_inputs)` in CI as a general test so this won't pass. 
In general if training data is different from inference, we'll just separate them out instead of reusing `example_inputs`?

Reply:
i ran into the same thing.. i worked around by passing example_inputs[:2] into model, which is ugly. I was about to delete my test that covers `model(*example_inputs)`.  I wasn't sure if that was even part of the intentional 'API' we're providing.

What's the difference between calling model(*inputs) vs calling m.eval() again? I think you explained that to me already.. 

I think we can also store 'self.train_inputs' and 'self.eval_inputs', so there is maximal flexibility?

Reply:
model(*inputs) vs calling m.eval() are the same in 99% cases. 
Here's an comment from @zdevito about the special case: 
```
the difference in the non-common cases would be that the typical eval is not an nn.Module but some other thing, and having the get_model ensures we return a nn.Module
```